netdata-installer.sh

```diff
@@ -578,7 +581,7 @@ bundle_libwebsockets() {
     return 0
   fi
 
-  if [ -z "$(command -v cmake)" ] ; then
+  if [ -z "$(command -v cmake)" ]; then
```

Comment:
Not sure where these lines came from but this is not the right way to test for the existence of a command on the system. We should isntead use:

```#!sh
if command -v "$command" >/dev/null 2>&1; then
  echo "$command exists"
fi
```